app/models/station.rb

```diff
@@ -0,0 +1,51 @@
+class Station < ActiveRecord::Base
+
+  validates :name,
+            :dock_count,
+            :city_id,
+            :installation_date,
+            :lat,
+            :long,
+            presence: true
+
+  validates :name, uniqueness: true
+
+  belongs_to :city
+  has_many :departure_trips, class_name: 'Trip', :foreign_key => :start_station_id
+  has_many :arrival_trips, class_name: 'Trip', :foreign_key => :end_station_id
+
+  def self.total_station_count
+    count
+  end
+
+  def self.average_bikes_per_station
+    average(:dock_count).to_i
+  end
+
+  def self.maximum_bikes_per_station
+    maximum(:dock_count)
+  end
+
+  def self.stations_with_maximum_bikes_available
+    where(dock_count:(maximum(:dock_count))).pluck(:name).join(", ")
+  end
+
+  def self.minimum_bikes_per_station
+    minimum(:dock_count)
+  end
+
+  def self.stations_with_minimum_bikes_available
+    where(dock_count:(minimum(:dock_count))).pluck(:name).join(", ")
+  end
+
+  def self.newest_station
+    station = Station.find_by(installation_date: (Station.maximum(:installation_date)))
```

Comment:
i'd separate the two pieces of logic you have in this method to be able to reuse it. you are first finding the station, then you are formatting it. you have the same line of code on line 48. let's make that it's own method and call it twice 👍 

because this has to do with formatting the station name and date, i might suggest putting it in a helper method!